CONTRIBUTING.md

```diff
@@ -35,14 +35,36 @@ Useful questions to answer if you're having problems include:
 ## Submitting code changes
 
 Every new feature and bugfix is expected to go through code review before inclusion.
-We use Github pull requests for code review.
+We use GitHub pull requests for code review.
+Use the checklist below to create a good pull request.
```

Comment:
Minor, but maybe a friendlier tone? "The checklist below contains some of our expectations, and will help you create a good pull request."